receiver/prometheusreceiver/internal/metricfamily.go

```diff
@@ -65,6 +69,7 @@ func newMetricFamily(metricName string, mc scrape.MetricMetadataStore, logger *z
 		groups:      make(map[uint64]*metricGroup),
 		name:        familyName,
 		metadata:    metadata,
+		exemplars:   make(map[uint64]pmetric.ExemplarSlice),
```

Comment:
Why having a different map? Can we use the same map and have pmetric.ExemplarSlice inside the group?

Reply:
Moved to metric group